packages/react/src/types.ts

```diff
@@ -100,7 +100,7 @@ export type CSSPseudos =
  * excess property check doesn't enable makers to circumvent the
  * system and pass in values they shouldn't.
  */
-export type CSSProperties = Readonly<CSS.Properties<string | number>>;
```

Comment:
Ah, yep. That'd do it. Surprised it didn't affect more!

Reply:
I assume it touched a lot of properties to be honest, just until our strict mode, we never needed the inference to work.